Increase size of options dialog to fit wider fonts and long translations

adipose opened the pull request:
Probably could use testing, but I programmatically resized the options dialog, and then tweaked a few pain points where things didn't fit in the past in Russian.

Also changed the Formats page to have the same layout in admin mode and regular mode--this plus the wider button allows me to get rid of dynamic shifting code, which I've commented out.

Also changed the text for "has not enough privileges" to something more English-sounding.

Fixed one messagebox that wasn't themed (Formats page).

adipose committed:
Increase size of options dialog to fit wider fonts and long translations

clsid2 commented:
Looks good! You can remove the commented out code. It is always still available in git history.

Some minor issues:
player > keys: list can be taller
playback > fullscreen: the "audio delay" column title in the table is truncated
external filters: bottom listbox can be taller
tweaks: jump distances section box could use full width like in the other option pages
advanced: the "name" column could be a bit wider by default to avoid truncated option names

adipose committed:
tweak a few sizes

adipose committed:
cleanup deprecated code

adipose commented:
> player > keys: list can be taller 

fixed.

> playback > fullscreen: the "audio delay" column title in the table is truncated

fixed.

> external filters: bottom listbox can be taller

fixed.  Was supposed to autosize but I missed the treectrls.

> tweaks: jump distances section box could use full width like in the other option pages

fixed.

> advanced: the "name" column could be a bit wider by default to avoid truncated option names

I made an adjustment here because the second column doesn't always fit the header (Russian).  But actually the columns were autosized before (just not for the header), so it should never have been cut off.  I can make it artificially wider if you think that would look better.
